### PR TITLE
Clean up unit tests

### PR DESCRIPTION
**Pull request by jaredsburrows**
@vanniktech Updating and clean up unit tests

### CODE REVIEW

File: src/test/groovy/com/vanniktech/android/apk/size/ApkSizePluginSpec.groovy [hunk 1]
```diff
@@ -1,52 +1,43 @@
...
-        exception.message == 'APK Size Plugin requires the Android Application or Library plugin to be configured'
+    def 'setup'() {
+        project = ProjectBuilder.builder().build()
     }
 
-    def 'notAndroidProject'() {
-        given:
-        Project project = ProjectBuilder.builder().build()
-
+    @Unroll
```
Comment (jaredsburrows) [inline src/test/groovy/com/vanniktech/android/apk/size/ApkSizePluginSpec.groovy:16]: `@Unroll` acts as a parametrized unit test.

File: src/test/groovy/com/vanniktech/android/apk/size/ApkSizeToolsSpec.groovy [hunk 2]
```diff
@@ -13,30 +13,30 @@ class ApkSizeToolsSpec extends Specification {
         Locale.setDefault(defaultLocale)
     }
 
-    def 'convertBytesToMegaBytes'() {
+    def 'convert bytes to mega bytes'() {
         expect:
         Locale.availableLocales.each { locale ->
             Locale.setDefault(locale)
 
-            assert '0.01' == ApkSizeTools.convertBytesToMegaBytes(10000)
```
Comment (jaredsburrows) [inline src/test/groovy/com/vanniktech/android/apk/size/ApkSizeToolsSpec.groovy:21]: No need for assert, `expect` block is the assertions.

**Review by vanniktech — APPROVED** [review]
Thank you @jaredsburrows and sorry that it took me that long

**Comment by jaredsburrows** [comment]
@vanniktech Anytime. We can start working on code coverage next!

**Pull request merged**